### PR TITLE
instance: fix several bugs

### PR DESCRIPTION
**Pull request by jma**
* Fixes item types type validation.
* Fixes validation message for patron phone number.
* Adds opening hours to the library of the 3th organisation.
* Fixes document detailed view for patrons with organisation without
pickup locations. Closes: #598.
* Fixes import document using EAN from the BNF server. Closes: #607.
* Improves logs to BNF document importation.

Co-Authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

## Why are you opening this PR?

- First workshop failed.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

File: rero_ils/modules/documents/views.py [hunk 1]
```diff
@@ -113,46 +111,76 @@ def cover(isbn):
...
+            'errors': {
+                'code': status_code,
+                'title': 'The EAN was not found on the BNF server.'
+            }
         }
-        return jsonify(response), 404
     # other errors
-    except Exception:
-        sys.stdout.flush()
+    except Exception as e:
```
Comment (rerowep) [inline rero_ils/modules/documents/views.py]: Recommendation from python: Use more verbose names for your variables for clarity. `e`--> `err` ?

File: rero_ils/modules/documents/views.py [hunk 2]
```diff
@@ -113,46 +111,76 @@ def cover(isbn):
...
+        status_code = 500
         response = {
-            'record': {}
+            'metadata': {},
+            'errors': {
+                'code': status_code,
+                'title': 'An unexpected error has been raise.',
+                'detail': 'Error: {}'.format(e)
```
Comment (rerowep) [inline rero_ils/modules/documents/views.py]: I prefer to use named format parameters:
`'Error: {err}'.format(err=err)`

File: rero_ils/modules/documents/views.py [hunk 3]
```diff
@@ -113,46 +111,76 @@ def cover(isbn):
...
+            }
         }
-        return jsonify(response), 500
+        current_app.logger.error(
+            '{}: {}'.format(
```
Comment (rerowep) [inline rero_ils/modules/documents/views.py]: See above.

File: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json [hunk 4]
```diff
@@ -562,7 +562,7 @@
       "title": "Authors",
       "description": "Author(s) of the resource. Can be either persons or organisations.",
       "type": "array",
-      "minItems": 0,
+      "minItems": 1,
```
Comment (rerowep) [inline rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json]: See: https://github.com/rero/rero-ils/pull/611 ??

Comment (jma) [reply rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json]: This should not be addressed by this PR. It has been commited buy error.

**Pull request merged**